perplexica/umbrel-app.yml

```diff
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: perplexica
+name: Perplexica
+tagline: An AI-powered answering engine
+icon: https://raw.githubusercontent.com/dennysubke/dennys-umbrel-app-gallery/main/denny-perplexica/logo.png
```

Comment:
We can remove the icon field, as we already have a process to submit the icon through the PR information while preparing the gallery assets once the app PR is approved.
```suggestion
```